### PR TITLE
feat: experiment result by course id and session id.

### LINKED ISSUE #100: feat: experiment result by course id and session id Endpoint
inquire on session from P.O

### PR DESCRIPTION
**Pull request by bynalab**
resolve #100

### CODE REVIEW

File: app/Http/Controllers/Api/ExperimentController.php [hunk 1]
```diff
@@ -54,7 +55,7 @@ public function create(Request $request)
         $experiment_intro = $request->get('experiment_intro');
         $experiment_goal = $request->get('experiment_goal');
         $experiment_mock = $request->get('experiment_mock');
-        $aparatus = $request->get('aparatus');
+        $apparatus = $request->get('apparatus');
```
Comment (Auwalms) [inline app/Http/Controllers/Api/ExperimentController.php:58]: Thanks for fixing this @bynalab

**Review by Auwalms — APPROVED** [review]
LGTM

**Comment by abdullahikawu** [comment]
Good

**Review by abdullahikawu — APPROVED** [review]
Great work

**Pull request merged**